tests/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ foreach(test_name adc1 adc2 casscf-fzc-sp casscf-semi casscf-sa-sp ao-casscf-sp
                   cc50 cc51 cc52 cc53 cc54 cc55 cc5a cc6 cc7 cc8 cc8a cc8b cc8c
                   cc9 cc9a cdomp2-1 cdomp2-2 cepa0-grad1 cepa0-grad2 cepa1
                   cepa2 cepa3 cepa4 cepa-module ci-multi cisd-h2o+-0 cisd-h2o+-1
-                  cisd-h2o+-2 cisd-h2o-clpse cisd-opt-fd cisd-sp cisd-sp-2
+                  cisd-h2o+-2 cisd-h2o-clpse cisd-opt-fd cisd-sp cisd-sp-2 cppe
```

Comment:
ah, this doesn't go here but instead the `cppe/` dir gets added like https://github.com/psi4/psi4/pull/1401/files#diff-3dea97423f83302e647a3bab62bf1157L157

Reply:
Don't know what line the link refers to :slightly_frowning_face: 
Could you specify a line number?

Reply:
https://github.com/psi4/psi4/blob/master/tests/CMakeLists.txt#L155-L161

Reply:
Done.